docs/source/API/core/policies/RangePolicy.rst

```diff
@@ -0,0 +1,144 @@
+| ExecutionSpace    | ``Serial``, ``OpenMP``, ``Threads``, ``Cuda``, ``HIP``, ``SYCL``, ``HPX`` | Specify the Execution Space to execute the kernel in. Defaults to ``Kokkos::DefaultExecutionSpace``.                                                    |
++-------------------+---------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Schedule          | ``Schedule<Dynamic>``, ``Schedule<Static>``                               | Specify scheduling policy for work items. ``Dynamic`` scheduling is implemented through a work stealing queue. Default is machine and backend specific. |
++-------------------+---------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| IndexType         | ``IndexType<int>``                                                        | Specify integer type to be used for traversing the iteration space. Defaults to ``int64_t``.                                                            |
++-------------------+---------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| LaunchBounds      | ``LaunchBounds<MaxThreads, MinBlocks>``                                   | Specifies hints to to the compiler about CUDA/HIP launch bounds.                                                                                        |
++-------------------+---------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| WorkTag           | ``SomeClass``                                                             | Specify the work tag type used to call the functor operator. Any arbitrary type defaults to ``void``.                                                   |
++-------------------+---------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Requirements
+~~~~~~~~~~~~
+
+Public Class Members
+--------------------
+
+Constructors
+~~~~~~~~~~~~
+ 
+.. cpp:function:: RangePolicy()
+    
+* Default Constructor uninitialized policy.
+
+.. code-block:: cpp
+
+   template<class ... InitArgs> 
+   RangePolicy(const int64_t& begin, const int64_t& end, const InitArgs ... init_args)
+
+* Provide a start and end index as well as optional arguments to control certain behavior (see below).
```

Comment:
needs indentation

Reply:
why not a cpp function? either maike all cpp::function or none

Reply:
Yes, you're right, I have to be consistent: I also said it was weird.

I did that because you can't use cpp::function with these particular templates: 
![image](https://user-images.githubusercontent.com/62652182/210387790-b54f86df-bf5a-4434-bdfe-0c4af7a6076a.png)

As you can see there are three small dots. This triggers an error and is not understood by the sphinx syntax, unfortunately.

For the moment I correct by being consistent and putting code-block everywhere.